backend-api/src/functions/asyncIssueBiometricCredential/asyncIssueBiometricCredentialHandler.ts

```diff
@@ -74,7 +90,88 @@ export async function lambdaHandlerConstructor(
   if (viewerKeyResult.isError) {
     throw new RetainMessageOnQueue("Failed to retrieve biometric viewer key");
   }
+  const submitterKey = viewerKeyResult.value;
+
+  const sessionResult = await dependencies.getBiometricSession(
```

Comment:
issue: could this const be named something idfferently so it's not confused with any session result when getting OUR session

```suggestion
  const biometricSessionResult = await dependencies.getBiometricSession(
```

Reply:
Updated naming